spec/factories/merge_requests.rb

```diff
@@ -1,7 +1,8 @@
 FactoryBot.define do
   factory :merge_request do
-    status { "incomplete" }
+    status { "Incomplete" }
```

Comment:
Why is this change needed? I feel like we should save the status as lowercase

Reply:
No reason really, will revert it